index.js

```diff
@@ -14,10 +14,45 @@ const { WorldView, Viewer } = require('prismarine-viewer/viewer')
 const pathfinder = require('mineflayer-pathfinder')
 const { Vec3 } = require('vec3')
 global.THREE = require('three')
+let highlightCube
 
 const maxPitch = 0.5 * Math.PI
 const minPitch = -0.5 * Math.PI
 
+async function moveHighlight (x, y, z) {
+  const size = 1
+  const offset = 0.005
+
+  const points = []
+  /* top */
+  points.push(new THREE.Vector3(x + size + offset, y + size + offset, z + size + offset))
+  points.push(new THREE.Vector3(x - offset, y + size + offset, z + size + offset))
+  points.push(new THREE.Vector3(x - offset, y + size + offset, z))
+  points.push(new THREE.Vector3(x + size + offset, y + size + offset, z - offset))
+  points.push(new THREE.Vector3(x + size + offset, y + size + offset, z + size + offset))
+
+  /* bottom and connectors */
+  points.push(new THREE.Vector3(x + size + offset, y - offset, z + size + offset))
+
+  points.push(new THREE.Vector3(x, y - offset, z + size + offset))
+  points.push(new THREE.Vector3(x, y + size + offset, z + size + offset)) // connector
+  points.push(new THREE.Vector3(x, y - offset, z + size + offset))
+
+  points.push(new THREE.Vector3(x, y - offset, z))
+  points.push(new THREE.Vector3(x, y + size + offset, z)) // connector
+  points.push(new THREE.Vector3(x, y - offset, z))
+
+  points.push(new THREE.Vector3(x + size + offset, y - offset, z))
+  points.push(new THREE.Vector3(x + size + offset, y + size + offset, z)) // connector
+  points.push(new THREE.Vector3(x + size + offset, y - offset, z))
+
+  points.push(new THREE.Vector3(x + size + offset, y - offset, z + size + offset))
+
+  const geometry = new THREE.BufferGeometry().setFromPoints(points)
+
+  highlightCube.geometry = geometry
```

Comment:
You are recreating the geometry each time you move the cube: it is not efficient.
Instead move it using position.set() like you did before.